src/core/UBPreferencesController.cpp

```diff
@@ -293,6 +423,21 @@ void UBPreferencesController::wire()
 
     // about tab
     connect(mPreferencesUI->checkSoftwareUpdateAtLaunchCheckBox, SIGNAL(clicked(bool)), settings->appEnableAutomaticSoftwareUpdates, SLOT(setBool(bool)));
+
+    // shortcut tab
+    connect(mPreferencesUI->shortcutTableView, SIGNAL(activated(const QModelIndex&)), this, SLOT(actionSelected(const QModelIndex&)));
```

Comment:
activated as a signal makes it weird, in UX terms. On Windows + Qt 6.9.1, for example, you can select each column independently, amd need to double-click for the recording input to become enabled. 

Should use clicked instead.